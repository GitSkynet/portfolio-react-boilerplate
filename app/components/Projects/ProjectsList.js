import styled from 'styled-components';

const ProjectsList = styled.ul`
  margin: 0 auto;
  list-style: none;
  padding: 0 0 9vmin 0;
`;

export default ProjectsList;
