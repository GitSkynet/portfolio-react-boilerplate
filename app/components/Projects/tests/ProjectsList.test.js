import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import 'jest-styled-components';

import ProjectsList from '../ProjectsList';

describe('<ProjectsList />', () => {
  it('should render and match the snapshot', () => {
    const renderedComponent = shallow(<ProjectsList />);
    expect(toJson(renderedComponent)).toMatchSnapshot();
  });
});
