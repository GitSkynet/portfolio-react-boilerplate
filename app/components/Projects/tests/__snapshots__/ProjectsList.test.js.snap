// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ProjectsList /> should render and match the snapshot 1`] = `
.c0 {
  margin: 0 auto;
  list-style: none;
  padding: 0 0 9vmin 0;
}

<ul
  className="c0"
/>
`;
